src/controller.rs

```diff
@@ -49,6 +51,9 @@ pub trait Controller : Send + Sync + Clone + Reflect + 'static {
     fn services_as_json(&self) -> Result<String, serde_json::error::Error>;
     fn get_http_root_for_service(&self, service_id: String) -> String;
     fn get_ws_root_for_service(&self, service_id: String) -> String;
+    fn get_config(&self, namespace: String, property: String, default: String)
+        -> String;
```

Comment:
nit: feel free to keep that at the end of the previous line. Looks more readable to me.